.changes/next-release/Bug Fix-ec18555e-d716-40b5-b8d1-54dad851fdc2.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "CodeWhisperer supports code generation for Iaac languages without external extensions"
```

should this be IaC? what does Iaac stand for?
```suggestion
	"description": "CodeWhisperer supports code generation for IaC languages without external extensions"
```